spokestack/extensions/webrtc/agc.pyx

```diff
@@ -0,0 +1,94 @@
+"""
+This module contains the AutomaticGainControl class implemented in Cython
+"""
+# distutils: sources = filter_audio/agc/analog_agc.c, filter_audio/agc/digital_agc.c
+# distutils: include_dirs = filter_audio/agc/include/
+cimport numpy as np
+cimport cagc
+
+
+MIC_MAX = 255
+MIC_TARGET = 180
+
+np.import_array()
+
+cdef class AutomaticGainControl:
+    cdef void*_agc
+    cdef int _sample_rate
+    cdef int _frame_width
+    cdef int _target_level_dbfs
+    cdef int _compression_gain_db
+    cdef int _limit_enable
+    cdef cagc.WebRtcAgc_config_t _config
+
+    def __dealloc__(self):
+        cagc.WebRtcAgc_Free(self._agc)
+
+    def __init__(self,
+                 sample_rate=16000,
+                 frame_width=20,
+                 target_level_dbfs=0,
```

Comment:
`target_level_dbfs` and `compression_gain_db` should have defaults that are consistent with android
https://github.com/spokestack/spokestack-android/blob/master/src/main/java/io/spokestack/spokestack/webrtc/AutomaticGainControl.java#L45